src/silo_api/database_directory_watcher.cpp

```diff
@@ -150,7 +155,7 @@ void silo_api::DatabaseDirectoryWatcher::checkDirectoryForData(Poco::Timer& /*ti
       }
    }
    SPDLOG_INFO(
-      "New database with version {} successfully loaded.",
+      "Did not load new database with version {} successfully.",
```

Comment:
what happens if new version is not loaded successfully? does it retry later?

Reply:
Yes, it will simply try again.

Reply:
Great, just out of curiosity, what's retry interval?

Reply:
Ah 2 seconds